### DISCUSSION TITLE
Channel stuck on PENDING CLOSE

### DISCUSSION THREAD

**Discussion by mathieu-gilardet** [lightningnetwork/lnd#7962] [Troubleshooting]
Hi LND,

I have an issue with a channel stuck on PENDING CLOSE, and would like some advice on how to proceed.

### Setup

* BTCPay: `1.11.4`
* RTL: `0.14.0` 
* LND: `v0.16.4-beta`

### Context / What I did:

- On RTL, I clicked on close channel, and set the fees as 3 target blocks.
- I've had a message saying 'backup successfull'
- But the channel remained on OPEN state, and went INACTIVE
- Reading, about inactive channels on your issues page, someone suggested to the restart the LND deamon, which i did.
- Then the channel went PENDING CLOSE, which is the current state.
- But the shown transactions hashes have not been published on the mempool.

So, before rushing everthing and making mistakes,

- Could you please advise me, on what you think is the best strategy to close the channel? From what I have read, I am guessing force close, but I would appreciate hearing it from you.

- If force close is the way, then how to issue the command?

here is the `--help` on `closechannel`: 

```
USAGE:
   lncli closechannel [command options] funding_txid [output_index]
```

I am affraid since I used an inbound Lightning Liquidy Provider, that a don't know the funding tx hash, and `pendingchannels` doesn't show it...

Any help would be greatly appreciated, thank you very much!

## Outputs

```
# lncli pendingchannels
{
    "total_limbo_balance": "25324955",
    "pending_open_channels": [
    ],
    "pending_closing_channels": [
    ],
    "pending_force_closing_channels": [
    ],
    "waiting_close_channels": [
        {
            "channel": {
                "remote_node_pub": "<my-lighting-node-id>",
                "channel_point": "9364182a16a7ba734e2851afe3216051e75000d152db9b68abb945c33dc2e40c:1",
                "capacity": "49964410",
                "local_balance": "25324955",
                "remote_balance": "24636098",
                "local_chan_reserve_sat": "499644",
                "remote_chan_reserve_sat": "499644",
                "initiator": "INITIATOR_REMOTE",
                "commitment_type": "ANCHORS",
                "num_forwarding_packages": "0",
                "chan_status_flags": "ChanStatusCoopBroadcasted|ChanStatusLocalCloseInitiator",
                "private": false
            },
            "limbo_balance": "25324955",
            "commitments": {
                "local_txid": "94e629347e4d01ab75c50b83fd1aaf69d6b4e4a0ffaa57828bed1e0ec69a8380",
                "remote_txid": "2a61d1396b2d7c88b658b5541e3236487d72ff5e0785f664f6c043d3aef16169",
                "remote_pending_txid": "",
                "local_commit_fee_sat": "2696",
                "remote_commit_fee_sat": "2696",
                "remote_pending_commit_fee_sat": "0"
            },
            "closing_txid": ""
        }
    ]
}
```

**Comment by guggero** [comment]
This can sometimes happen if your node and your peer can't agree on a closing fee. You can try again by disconnecting from that peer and re-connecting, then attempt to normally close the channel.
If that doesn't work, the `channel_point` shown in the closing channel list is the funding transaction ID and the output index, separated by a colon. So if you want to force close, the command would in your case be: `lncli closechannel --force --funding_txid 9364182a16a7ba734e2851afe3216051e75000d152db9b68abb945c33dc2e40c --output_index 1`

**Reply by mathieu-gilardet** [reply]
Thank you guggero for your fast reply,
- I've tried disconnecting then reconnecting peer, but the channel was still in `pending close` state, so I did not see any way to retry a soft close.
- Then I've tried a force close as advised: I am happy to see my tx on the BC.

I had seen on some posts suggestions to use `channel_point` as `funding_txid` but wasn't sure... Thank you very much for your confirmation and in general your assistance!

Please have a nice day :)
